### DISCUSSION TITLE
Bug in Calendar Event.get_body_text / get_body_soup

### DISCUSSION THREAD

**Discussion by Amitielle** [O365/python-o365#883] [General]
I never really worked on GitHub so pardon me please if I'm wrong here. I just found a little bug in Calendar Events methods get_body_text and get_body_soup and thought it would be nice to let you know:

The comparison of self.body_type with 'HTML' doesn't work if body_type is 'html' and therefore this check fails altough it's still HTML.

```
def get_body_text(self):
        """ Parse the body html and returns the body text using bs4

        :return: body text
        :rtype: str
        """
        if self.body_type != 'HTML':
            return self.body

        try:
            soup = bs(self.body, 'html.parser')
        except RuntimeError:
            return self.body
        else:
            return soup.body.text
```

**Comment by alejcas** [comment]
This wall already fixed:

```python
    def get_body_soup(self):
        """ Returns the beautifulsoup4 of the html body

        :return: BeautifulSoup object of body
        :rtype: BeautifulSoup
        """
        if self.body_type.upper() != 'HTML':
            return None
        else:
            return bs(self.body, 'html.parser')
```

Update to last version please

**Reply by Amitielle** [reply]
That's strange, I installed via pip just two days ago and assumed I got the latest version. Anyways, thanks for the reply I'll make sure to get the latest version in place.
